RUFAS/biophysical/animal/pen.py

```diff
@@ -693,9 +676,59 @@ def get_manure_streams(self) -> dict[int, list[dict[str, ManureStream]]]:
             )
             if manure_stream.pen_manure_data is not None:
                 manure_stream.pen_manure_data.set_first_processor(str(stream.get("first_processor")))
-            animal_manure_streams.append({str(stream.get("stream_name")): manure_stream})
+            manure_stream = self._apply_bedding(manure_stream, str(stream.get("bedding_name")))
+            stream_name = f"{str(stream.get('stream_name'))}_{self.animal_combination.name}_PEN_{self.id}"
+            animal_manure_streams[stream_name] = manure_stream
+
+        return animal_manure_streams
 
-        return {self.id: animal_manure_streams}
+    def _apply_bedding(self, manure_stream: ManureStream, bedding_name: str) -> ManureStream:
+        """
+        Applies bedding to the given manure stream.
+
+        Parameters
+        ----------
+        manure_stream : ManureStream
+            The manure stream object.
+        bedding_name : str
+            The name of the bedding, this should correspond to a key in the `beddings` attribute.
+
+        Returns
+        -------
+        ManureStream
+            A new ManureStream object with updated attributes reflecting the impact of the applied bedding.
+
+        """
+        bedding = self.beddings[bedding_name]
+        if manure_stream.pen_manure_data is not None:
+            num_animals = manure_stream.pen_manure_data.num_animals
```

Comment:
```suggestion
        if manure_stream.pen_manure_data is None:
            raise ValueError("pen_manure_data must be set to apply bedding.")
        num_animals = manure_stream.pen_manure_data.num_animals
```
If this is just to avoid the mypy error, maybe this would be more explicit about what's happening and have a solution if there is None pen_manure_data.